main.nf

```diff
@@ -53,6 +53,7 @@ def helpMessage() {
 	  --min_samples [int]           Filtering low prevalent features from the feature table (default: 1)                   
 
 	Cutoffs:
+          --double_primer               Cutdapt will be run twice, first to remove reads without primers (default), then a second time to remove reads that erroneously contain a second set of primers, not to be used with "--retain_untrimmed"
```

Comment:
> , not to be used with "--retain_untrimmed"

As user, I interpret this in a way that I may not use the parameter '--retain_untrimmed' with `--double_primer`. And this is not true, is it? I read the code below that the second round of cutadapt will ignore '--retain_untrimmed', so it doesn't matter.

Reply:
As I've setup the if-else statement - to also evaluate whether "discard untrimmed" is set or not - it won't be compatible. However, it would be possible to change that - but I can't really see that anyone that wants to keep sequences without primers then would like to discard sequences that still contain primers? I thought that "--retain_untrimmed" is for special cases, and not really recommended.

Reply:
I see. "--retain_untrimmed" exists because the pipeline should not prohibit to analyze reads which had their primers removed previously (but it is indeed not recommended as you mentioned). However, I imagine that also in this cases where previously primers were removed (e.g. a first round of cutadapt), a second primer sequence of a faulty PCR could be retained? But in this case the first round of cutadapt in ampliseq would be the actual second round of primer removal and would need to discard any remaining reads with primers.

Ok, that maybe unnecessarily complicates the matter.

Instead, I am now recommending to add an error message to the pipeline when `--double_primer` and `--retain_untrimmed` are set simultanious. For example in the section [here](https://github.com/nf-core/ampliseq/blob/70b23f4710094df9299e5ab2d18a1d412bbd4d04/main.nf#L181).

Reply:
To be clear, should I keep it as I set it up and just add the error message?

Reply:
Yes, I think that should do it.
I thought something like
```
if (params.double_primer && params.retain_untrimmed) { 
	exit 1, "Incompatible parameters --double_primer and --retain_untrimmed cannot be both set."
}
```